### PR TITLE
OCP-23395- Deleted catalog registry pods and verify if them are recreated automatically

### PR DESCRIPTION
**Pull request by bandrade**
Migrated from: https://github.com/openshift/origin/pull/24353

cc: @cuipinghuo @scolange @chengzhang1016 @emmajiafan @jianzhangbjz @tbuskey @kuiwang02

### CODE REVIEW

File: test/extended/operators/olm.go [hunk 1]
```diff
@@ -315,4 +315,34 @@ var _ = g.Describe("[Feature:Platform] an end user use OLM", func() {
 		}
 	})
 
+	// OCP-23395 - Deleted catalog registry pods and verify if them are recreated automatically
+	// author: bandrade@redhat.com
+	g.It("OLM-Low-OCP-23395-Recreate catalog registry pods and if they are deleted", func() {
```
Comment (jianzhangbjz) [inline test/extended/operators/olm.go:320]: The importance of OCP-23395 should be `Medium`, not `Low`.

**Comment by bandrade** [comment]
migrated to https://github.com/openshift/openshift-tests-private/pull/33

**Pull request closed without merging**